api/endpoint-tests/auth/login.js

```diff
@@ -9,6 +9,18 @@ tap.test('login endpoint | /auth/login', async loginTest => {
     invalidTest.equal(response.statusCode, 400, 'gives a 400 status code');
   });
 
+  loginTest.test('proper response headers', async headerTest => {
+    const { response } = await request.post(url);
+    const { headers } = response;
+
+    headerTest.equal(headers.vary, 'Origin', 'vary origin');
+    headerTest.equal(
+      headers['access-control-allow-credentials'],
+      'true',
+      'allow credentials'
+    );
```

Comment:
Nice!  I'm curious if these two asserts should be part of all the login tests, but that's probably overkill.